### DISCUSSION TITLE
OpenPDF equivalent in nodejs

### DISCUSSION THREAD

**Discussion by RakeshMaddala** [LibrePDF/OpenPDF#968] [General]
Hello, I am looking for OpenPDF equivalent for nodejs environment. Please provide any suggestions.

Sincerely,
Thank you
Rakesh

**Comment by fellmann** [comment]
the most common pure js pdf implementation would be https://pdfkit.org/

**Comment by shaikhsohel445** [comment]
https://github.com/Hopding/pdf-lib

try this one
